client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -270,7 +270,7 @@ public RssShuffleManager(SparkConf conf, boolean isDriver) {
     return getReader(handle, 0, Integer.MAX_VALUE, startPartition, endPartition, context, metrics);
   }
 
-  @Override
+  // remove @Override, because Spark 2.3 does not have this interface
```

Comment:
This is Spark3 client.... This comment has problems.

Reply:
Fixed